.dockerignore

```diff
@@ -10,11 +10,5 @@ node_modules/
 .prettierrc.yml
 DEVELOPERS.md
 lerna-debug.log
-modules/*/
-!modules/express/*
-modules/express/dist/
-modules/express/test/
-modules/express/node_modules/
-modules/express/.nyc_output/
-modules/express/.gitignore
-modules/express/.npmignore
+modules/**/node_modules
```

Comment:
Would it make sense to move the dockerignore and Dockerfile to the express module?

Reply:
we should also still ignore the other files like `.nyc_output`, `.gitignore`, `.npmignore` etc

Reply:
Okay if we don't wanna copy them over during the build stage, I'll put them back. However, they are not included in the final image.